src/PhpParser/ValueObject/StmtsAndTokens.php

```diff
@@ -10,7 +10,7 @@ final class StmtsAndTokens
 {
     /**
      * @param Stmt[] $stmts
-     * @param mixed[] $tokens
+     * @param array<int, array{int, string, int}|string> $tokens
```

Comment:
for note, on downgrade, it may cause changed back to mixed, see issue: 

- https://github.com/rectorphp/rector/issues/8004

so that issue need to be resolved first